jekyll/_cci2/quickstart-javascript.md

```diff
@@ -54,42 +54,43 @@ jobs:
   build:
     working_directory: ~/cci-demo-react
     docker:
-      image: node:7.4.0
+      - image: node:7.4.0
     steps:
-      - shell:
-        name: Pre-Dependencies
-        command: mkdir ~/artifacts
+      - run:
+          name: Pre-Dependencies
+          command: mkdir /home/ubuntu/cci-demo-react/artifacts
```

Comment:
Should we change this back to `~/cci-demo-react/artifacts`? Or do we want to leave the `/home/ubuntu`?